### ISSUE TITLE
CI?

### ISSUE THREAD

**Issue by utoni** [yrutschle/sslh#343]
What about CI e.g. Github Actions?

I would like to see that `sslh` builds/runs on different platforms as well as the "unit" tests.

**Comment by yrutschle** [comment]
I agree in principle, though I think there is major (but necessary) work on the test suite.

**Comment by oliv3r** [comment]
@yrutschle I think with the docker container generation, we have the first bits for a CI in place. We can also now leverage that, in running the container, and tests with/within that.
